support.js

```diff
@@ -303,8 +304,12 @@ if (async.length) {
 }
 
 function onLoaded(fn) {
-  if (document.readyState == "loading") {
-    document.addEventListener('DOMContentLoaded', fn);
+  // IE 9 fires DOMContentLoaded, and enters the "interactive"
+  // readyState, before document.body has been initialized, so wait
+  // for window.load
+  if (document.readyState == "loading" ||
+      document.readyState == "interactive") {
```

Comment:
Please use single quotes on these ready states.  I was breaking our own style rules in the original.  :-)